### PR TITLE
Navigation

### LINKED ISSUE #3: Create a navigation menu
In this change,  create the navigation menu for the eduleaening and make it responsive to view in mobiles as well as desktop

### PR DESCRIPTION
**Pull request by motuncoded**
In this change, I create the navigation menu for the edulearning and made responsive to view in mobiles as well as desktop

closes #3

### CODE REVIEW

File: src/app/blog/page.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (Asylcreek) [inline src/app/blog/page.tsx:1]: You do not need to import React

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (Asylcreek) [inline src/app/about/page.tsx:1]: You do not need to import React

File: src/app/categories/page.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (Asylcreek) [inline src/app/categories/page.tsx:1]: You do not need to import React

File: src/app/components/navigation/navbar/page.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <PiBookOpenUserFill size="22" className="text-[var(--accent-color)] mr-2" />
+    <h1 className="text-[1.38rem]">edulearning</h1>
+  </header>
+);
+
+interface NavItemProps {
+  href: string;
+  children: ReactNode;
+}
+const NavItem: React.FC<NavItemProps> = ({ href, children }) => {
```
Comment (Asylcreek) [inline src/app/components/navigation/navbar/page.tsx:20]:
```suggestion
const NavItem = ({ href, children }: NavItemProps) => {
```

File: src/app/components/navigation/navbar/page.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      <NavItem href="/contact">Contact</NavItem>
+      <NavItem href="/blog">Blog</NavItem>
+    </ul>
+  </nav>
+);
+
+export default function Navbar() {
+  const [menu, setMenu] = useState(false);
+  const toggleMenu = () => {
+    setMenu(!menu);
```
Comment (Asylcreek) [inline src/app/components/navigation/navbar/page.tsx:49]: Use this updater function style instead

File: src/app/contact/page.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (Asylcreek) [inline src/app/contact/page.tsx:1]: You do not need to import React

File: src/app/contact/page.tsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+function page() {
```
Comment (Asylcreek) [inline src/app/contact/page.tsx:3]: The convention is to have React component names in PascalCase

File: src/app/categories/page.tsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+function page() {
```
Comment (Asylcreek) [inline src/app/categories/page.tsx:3]: The convention is to have React component names in PascalCase

File: src/app/blog/page.tsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+
+function page() {
```
Comment (Asylcreek) [inline src/app/blog/page.tsx:3]: The convention is to have React component names in PascalCase

File: src/app/about/page.tsx [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+
+function page() {
```
Comment (Asylcreek) [inline src/app/about/page.tsx:3]: The convention is to have React component names in PascalCase

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/app/components/navigation/navbar/page.tsx (hunk 4 above)
Comment (motuncoded) [reply src/app/components/navigation/navbar/page.tsx:20]: ok

**Review by Asylcreek — APPROVED** [review]

**Pull request merged**